.doctor-rst.yaml

```diff
@@ -72,10 +72,10 @@ rules:
         min_version: '6.0'
 
     deprecated_directive_major_version:
-        major_version: 6
+        major_version: 7
 
     deprecated_directive_min_version:
-        min_version: '6.0'
```

Comment:
Let's revert for now, to keep the builds green until 7.0 is released, otherwise it's a lot of work on every upmerge from a new deprecation/feature.

Thanks

Reply:
Here you go :slightly_smiling_face: